image-template/README.md

```diff
@@ -0,0 +1,2 @@
+# Custom image as Packer base
+This template demonstrates use a custom image as the starting point for the Packer template
```

Comment:
Do you think it's worth to add some extra details about what are the different properties inside the `image-template.json` file? Might be useful to describe in the readme the different input variables and highlight the parts of the json that are relevant for a custom image? Like you did for the shared image gallery.

Reply:
You can actually ignore my previous comment, everything is well documented into the main README :)

Reply:
I just forgot about this one. It's still light though because there's not much to say.